### PR TITLE
DOC-4272: Add documentation for the possibility to comment directly f…

### PR DESCRIPTION
**Pull request by mzorai**
…rom activity stream in mobile

### CODE REVIEW

File: docs/Features.rst [hunk 1]
```diff
@@ -92,6 +96,7 @@ with your social intranet in a familiar environment.
    :width: 10.00000cm
 .. |image4| image:: images/mobile/documents_share.png
 .. |image5| image:: images/mobile/mobile_upload_files.png
+.. |image6| image:: images/mobile/commentAS_mobile.png
```
Comment (boubaker) [inline docs/Features.rst:99]: The file commentAS_mobile.png is missing

Comment (mzorai) [reply docs/Features.rst:99]: I added it right now, thanks

**Comment by mzorai** [comment]
The rendering of the PR could be viewed through this link https://docs-preprod.exoplatform.org/en/preproduction/Features.html#news-feed

**Comment by mzorai** [comment]
The rendering of the PR could be viewed through this link https://docs-preprod.exoplatform.org/en/preproduction/Features.html#news-feed

**Review by hghorbel — APPROVED** [review]

**Pull request merged**